vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/create/ProjectCreationDialog.java

```diff
@@ -63,12 +61,12 @@ public class ProjectCreationDialog extends Dialog {
   public final ComboBox<PersonReference> responsiblePerson = new ComboBox<>(
       "Project Responsible (optional)");
   public final ComboBox<PersonReference> projectManager = new ComboBox<>("Project Manager");
-  private final Button createButton = new Button("Create");
-  private final Button cancelButton = new Button("Cancel");
+
   private final Handler handler;
 
   public ProjectCreationDialog(ExperimentalDesignSearchService experimentalDesignSearchService) {
-
+    super("Create", "Cancel");
```

Comment:
NIT: I don't think this is very readable. 
Also what happens if no values are provided? Do we want to allow nonlabeled buttons? 
```suggestion
    super("Create", "Cancel");
```

Reply:
What would be a way to change that?